templates/CRM/Contact/Form/Search/Advanced.tpl

```diff
@@ -13,7 +13,7 @@
 
   {include file="CRM/Contact/Form/Search/Intro.tpl"}
 
-  <details class="crm-advanced_search_form-accordion" open>
+  <details class="crm-advanced_search_form-accordion" {if $rows}{else}open{/if} >
```

Comment:
This should work, but it would be simpler to say
```suggestion
  <details class="crm-advanced_search_form-accordion" {if !$rows}open{/if} >
```

Reply:
Thanks - I thought I'd tried that but must have been`{if !$rowsEmpty}`